templates/vue/src/components/FileUpload.vue

```diff
@@ -35,20 +35,6 @@
           @change="uploadFile"
         ></b-form-file>
       </b-col>
-      <b-col sm="1" class="divider">
-        <h6 class="text-muted">OR</h6>
-      </b-col>
-      <b-col class="pl-0">
-        <b-form-input
-          name="text-input"
-          :placeholder="placeholder"
-          :value="value"
-          :data-qa="inputTestId"
-          :disabled="isUploading"
-          required
-          @input="$emit('input', $event)"
-        />
-      </b-col>
```

Comment:
You should keep this in and hide/show it using a prop. We still need it for everything else (e.g. Video URLs, External Links, etc.)

Reply:
Thanks for the catch